static/app/views/dashboards/manage/dashboardGrid.spec.tsx

```diff
@@ -271,10 +278,17 @@ describe('Dashboards - DashboardGrid', function () {
         rowCount={3}
       />
     );
+    renderGlobalModal();
 
     await userEvent.click(screen.getAllByRole('button', {name: /dashboard actions/i})[1]);
     await userEvent.click(screen.getByTestId('dashboard-duplicate'));
 
+    expect(createMock).not.toHaveBeenCalled();
```

Comment:
```suggestion
    expect(postMock).not.toHaveBeenCalled();
```
Same note as other comment